src/components/CompositeBar/Item/Item.scss

```diff
@@ -103,6 +116,18 @@ $block: '.#{variables.$ns}composite-bar-item';
         height: 100%;
         cursor: pointer;
 
+        background: none;
```

Comment:
Can we move the duplicate code into a mixin?

Reply:
Great catch, thank you! Fixed.